src/patternfly/components/Popover/popover.scss

```diff
@@ -56,9 +56,9 @@
 
   // Title text
   --#{$popover}__title-text--Color: var(--pf-t--global--text--color--regular);
-  --#{$popover}__title-text--FontFamily: var(--pf-t--global--font--family--body);
-  --#{$popover}__title-text--FontSize: var(--pf-t--global--font--size--body--lg);
   --#{$popover}__title-text--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$popover}__title-text--FontSize: var(--pf-t--global--font--size--heading--xs);
```

Comment:
Since this is using a body font, I'd think we wouldn't want to use the heading font-size? Technically it should work just fine, but AFAIK we aren't mixing the body and heading tokens. The equivalent in the body font-size tokens would be `--pf-t--global--font--size--body--lg`.